backend/dataall/modules/dataset_sharing/services/dataset_alarm_service.py

```diff
@@ -19,9 +19,7 @@ def trigger_table_sharing_failure_alarm(
             target_environment: Environment,
     ):
         log.info('Triggering share failure alarm...')
-        subject = (
-            f'ALARM: DATAALL Table {table.GlueTableName} Sharing Failure Notification'
-        )
+        subject = f'ALARM: DATAALL Sharing Failure Notification for Table {table.GlueTableName}'[:100]
```

Comment:
I would fix all of these to:

- remove ALARM: (don't think it's useful)
- rename DATAALL to data.all
- not use word notification and change to more concise version: "data.all share failure for table {table.GlueTableName}"

Reply:
Sure